src/argilla/client/feedback/dataset/base.py

```diff
@@ -306,7 +311,12 @@ def unify_responses(
     def prepare_for_training(
         self,
         framework: Union[Framework, str],
-        task_mapping: TrainingTaskMappingForTextClassification,
+        task: Union[
+            TrainingTaskForTextClassification,
+            TrainingTaskForSupervisedFinetuning,
+            TrainingTaskForRewardModelling,
```

Comment:
As the user may directly interact with e.g. `TrainingTask.for_text_classification`, shouldn't we shorten these names?

e.g. `TrainingTaskForDirectPreferenceOptimization` -> `TrainingTaskForDPO`

Reply:
I'm down for this. @davidberenstein1957 how do you feel about it? We'd have DPO, SFT, RM, (and later PPO).